### PR TITLE
Add of background, description and contribute to documentation

### PR DESCRIPTION
**Pull request by christoffergranberg**
Update of documentation regarding background and contribution requests

### CODE REVIEW

File: docs/source/user/background.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+patient size and the geometrical factors that greatly influence skin dose.
+
+A directly available metric for _quality assurance_ (QA) of patient dose is the cumulative air kerma. 
+This exposure value is referenced to a specific location, the _patient entrance reference point_ 
+(PERP). This reference point is located along the central X-ray beam, commonly but not always at a 
+distance of 15 cm from the isocenter in the direction of the X-ray tube. The cumulative air kerma 
+is available on most interventional C-arm X-ray equipment since 2006, when it became mandated by the 
+FDA. However, the cumulative air kerma also has known limitations e.g., patient skin positioning in 
+relation to the PERP, beam attenuation of the patient table and pad, influence of scattered radiation 
+etc. The currently most useful metric to indicate a risk of erythema, epilation or greater skin injury
```
Comment (BwKodex) [inline docs/source/user/background.md:17]: Should this be dated so as not to risk it being missleading in the future if we don't get to changing the text in time?

File: docs/source/user/background.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+relation to the PERP, beam attenuation of the patient table and pad, influence of scattered radiation 
+etc. The currently most useful metric to indicate a risk of erythema, epilation or greater skin injury 
+that also include actionable information is the _peak skin dose_ (PSD), which gives the largest dose to 
+a region of skin.
+
+Compared to other X-ray modalities, the greatest challenge in estimating patient dose metrics may be 
+found for C-arm fluoroscopy equipment, particularly for FGI procedures. This is due to complex 
+geometries, accuracy in representing patient position and dealing with uncertainty in metrics. There 
+are many different solutions that can be used for estimating skin dose, including commercial, 
+non-commercial and open-source. Many of these solutions are great tools but does not offer the user
```
Comment (BwKodex) [inline docs/source/user/background.md:25]: do

File: docs/source/user/background.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+a region of skin.
+
+Compared to other X-ray modalities, the greatest challenge in estimating patient dose metrics may be 
+found for C-arm fluoroscopy equipment, particularly for FGI procedures. This is due to complex 
+geometries, accuracy in representing patient position and dealing with uncertainty in metrics. There 
+are many different solutions that can be used for estimating skin dose, including commercial, 
+non-commercial and open-source. Many of these solutions are great tools but does not offer the user 
+a chance to analyze the effect of all the variables, add functionality and improve uncertainties with 
+in-house measurements. **P**y**S**kin**D**ose aims to be an open-source tool for estimation of PSD 
+with full transparency and used adoption functionality.
```
Comment (BwKodex) [inline docs/source/user/background.md:28]: Iäm not quite sure I understand the last part of this sentence

File: docs/source/user/background.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+follow-up. Metrics such as fluoroscopy time, air kerma-area product, or the number of _digital 
+subtraction angiography_ (DSA) runs have historically been used for radiation monitoring of patient 
+exposure. These metrics lack correlation to X-ray beam quality, variable pulse rates, dose rates, 
+patient size and the geometrical factors that greatly influence skin dose.
+
+A directly available metric for _quality assurance_ (QA) of patient dose is the cumulative air kerma. 
+This exposure value is referenced to a specific location, the _patient entrance reference point_ 
+(PERP). This reference point is located along the central X-ray beam, commonly but not always at a 
+distance of 15 cm from the isocenter in the direction of the X-ray tube. The cumulative air kerma 
+is available on most interventional C-arm X-ray equipment since 2006, when it became mandated by the
```
Comment (BwKodex) [inline docs/source/user/background.md:14]: has been

Comment (BwKodex) [inline docs/source/user/background.md:14]: Remove unnecessary comma after 2006

File: docs/source/user/background.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+subtraction angiography_ (DSA) runs have historically been used for radiation monitoring of patient 
+exposure. These metrics lack correlation to X-ray beam quality, variable pulse rates, dose rates, 
+patient size and the geometrical factors that greatly influence skin dose.
+
+A directly available metric for _quality assurance_ (QA) of patient dose is the cumulative air kerma. 
+This exposure value is referenced to a specific location, the _patient entrance reference point_ 
+(PERP). This reference point is located along the central X-ray beam, commonly but not always at a 
+distance of 15 cm from the isocenter in the direction of the X-ray tube. The cumulative air kerma 
+is available on most interventional C-arm X-ray equipment since 2006, when it became mandated by the 
+FDA. However, the cumulative air kerma also has known limitations e.g., patient skin positioning in
```
Comment (BwKodex) [inline docs/source/user/background.md:15]: Add comma before _e.g._

File: docs/source/user/description.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# What is PySkinDose?
+
+PySkinDose is an open-source Python™ package for RDSR-based skin dose estimation. This system 
+translates air kerma at the PERP for all irradiation event to estimate entrance skin dose for
```
Comment (BwKodex) [inline docs/source/user/description.md:4]: events

File: docs/source/user/description.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+# What is PySkinDose?
+
+PySkinDose is an open-source Python™ package for RDSR-based skin dose estimation. This system 
+translates air kerma at the PERP for all irradiation event to estimate entrance skin dose for 
+all irradiated surfaces on computational phantoms. The phantoms are oriented with the X-ray 
+source by geometric parameters found within the RDSR, and conversion from air Kerma at the PERP 
+to skin dose is further supported by correction factors found in literature, as well as a limited 
+number of in-clinic measurements (e.g., validation of the coordinate system, table and pad
```
Comment (BwKodex) [inline docs/source/user/description.md:8]: remove parenthesis and add comma before _e.g._

File: docs/source/user/description.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+# What is PySkinDose?
+
+PySkinDose is an open-source Python™ package for RDSR-based skin dose estimation. This system 
+translates air kerma at the PERP for all irradiation event to estimate entrance skin dose for 
+all irradiated surfaces on computational phantoms. The phantoms are oriented with the X-ray
```
Comment (BwKodex) [inline docs/source/user/description.md:5]: This is a long sentence. Could we split it in two to make it more easily readable?

File: docs/source/user/contribute.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+# Contributions and Collaborations
+
+PySkinDose is open-source and available for download from the project repository. Contributions and
```
Comment (BwKodex) [inline docs/source/user/contribute.md:3]: and PyPI

File: docs/source/user/contribute.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+of examples:
+
+- New functions for more accurate patient skin dose estimation
+- Added calculation of uncertainties
+- Suggestions of further development
+- Data for implementation of new/other FGI devices
+- Recommendations of new literature about patient skin dose estimation
+- Alteration of computational phantoms
+- Reviews of user friendliness
+- Etc.
```
Comment (BwKodex) [inline docs/source/user/contribute.md:14]: Do we need this bullet-point when the sentence above says _Listed below are a couple of examples:_ ?

File: docs/source/user/contribute.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
+# Contributions and Collaborations
+
+PySkinDose is open-source and available for download from the project repository. Contributions and
+collaborations for further development of PySkinDose is greatly appreciated. Listed below is a couple
```
Comment (BwKodex) [inline docs/source/user/contribute.md:4]: are

File: docs/source/user/background.md (hunk 1 above)
Comment (christoffergranberg) [reply docs/source/user/background.md:17]: added "year 2022"

File: docs/source/user/background.md (hunk 3 above)
Comment (christoffergranberg) [reply docs/source/user/background.md:28]: used = user

File: docs/source/user/background.md (hunk 2 above)
Comment (christoffergranberg) [reply docs/source/user/background.md:25]: ok

File: docs/source/user/description.md (hunk 7 above)
Comment (christoffergranberg) [reply docs/source/user/description.md:8]: ok

File: docs/source/user/description.md (hunk 8 above)
Comment (christoffergranberg) [reply docs/source/user/description.md:5]: moved "irradiation event" to the prior sentance and revised for a shorter sentance.

File: docs/source/user/contribute.md (hunk 9 above)
Comment (christoffergranberg) [reply docs/source/user/contribute.md:3]: ok

File: docs/source/user/contribute.md (hunk 10 above)
Comment (christoffergranberg) [reply docs/source/user/contribute.md:14]: Fair point. Removed "etc."

File: docs/source/user/contribute.md (hunk 11 above)
Comment (christoffergranberg) [reply docs/source/user/contribute.md:4]: ok

**Review by BwKodex — APPROVED** [review]

**Pull request merged**